.github/README.md

```diff
@@ -16,7 +16,7 @@ Oppia Android is written using Kotlin and Java, and leverages Bazel, databinding
 
 ## Installation
 
-Please refer to the [installation wiki page](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#install-oppia-android) for full instructions.
+Please refer to the [Installing-Oppia-Android](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android) page for full instructions on how to install Oppia-Android on local machine.
```

Comment:
`[Installing-Oppia-Android]` should probably use spaces between the words, rather than hyphens.

That said, I kind of prefer to keep it as-is, unless we also modify the Installation page to point back out to the general contribution instructions. What do you think? Otherwise we run into the issue of folks skipping those and then leaving random comments on issues (as often happens in Oppia Web). (We might want to fix the flow on Oppia Web as well so that everyone starts from the general instructions...)

Reply:
@seanlip 
Regarding hyphens i will implement the use of spaces from now.
Regarding the wiki i think that firstly we should update the flow of **_readme.md_** file for both _** web and android**_ with respect to the new contributors.
what do you think.
I think the welfare team is thinking of redesigning the docs but it may take time to first make a decision and then implementation. so firstly the readme.md file must be updated.

Reply:
I would only make the updates to the readme file that you are suggesting if the installation page points back to the general contribution instructions. Otherwise this just creates a path for folks to skip that entirely on onboarding.

Reply:
Will it be okay if we add the link to the general contribution instructions page at the end of the installation page.

Reply:
done